perf/src/main/java/tachyon/perf/benchmark/Operators.java

```diff
@@ -143,7 +143,7 @@ public static long readSingleFile(PerfFileSystem fs, String filePath, int buffer
     byte[] content = new byte[bufferSize];
     InputStream is = fs.open(filePath, readType);
     int onceLen = is.read(content);
-    while (onceLen != -1) {
+    while (onceLen > 0) {
```

Comment:
In this case, if the read returns 0, is it guaranteed to be done reading the file?